src/server/Main.cpp

```diff
@@ -97,15 +97,12 @@ int main() {
         typeid(Components::Gun),
         typeid(Components::Damage),
         typeid(Components::DeathRange),
-        typeid(Components::Scale),
+        // typeid(Components::Scale),
     };
 
     try {
-        engine.registerComponent<Components::Visible>("./plugins/bin/components/", "Visible");
-        engine.registerComponent<Components::Health>("./plugins/bin/components/", "Health");
-        engine.registerComponent<Components::Collider>("./plugins/bin/components/", "Collider");
-
-        engine.loadSystems("./src/server/configServer.cfg");
+        
+        std::cerr << ntohl(1291845632) << std::endl;
```

Comment:
debug left on purpose ?

Reply:
Nope, totally forgot about this one! Sorry

Reply:
No problem, as long as it is removed in the end :saluting_face: